README.md

```diff
@@ -13,7 +13,7 @@ Python package and commandline tool to evaluate the similarity between two image
 
 ## Installation
 
-Supports Python >=3.6.
+Supports Python >=3.8.
```

Comment:
@DylanBartels 
ques: why are we setting the min. python requirement as 3.8?

Reply:
I tried using 3.7 locally.
Since we are pinning sci-kit image to 0.20, we won't be able to support python3.7 (Unless, we pin down sci-kit image).

`The current project's Python requirement (3.7.16) is not compatible with some of the required packages Python requirement: scikit-image requires Python >=3.8, so it will not be satisfied for Python 3.7.16. Because scikit-image (0.20.0) requires Python >=3.8`

Reply:
Do you see a problem that we are not supporting 3.7?